core/use-swr.ts

```diff
@@ -511,6 +511,8 @@ export const useSWRHandler = <Data = any, Error = any>(
       return
     }
 
+    const isKeyHasActiveInstance =
```

Comment:
IMO, the name of `isKeyHasActiveInstance` might be a bit unclear to me. Is `hasRevalidator` clearer?